boa/src/bytecompiler.rs

```diff
@@ -38,6 +39,9 @@ struct JumpControlInfo {
     kind: JumpControlInfoKind,
     breaks: Vec<Label>,
     try_continues: Vec<Label>,
+    in_catch: bool,
```

Comment:
At some point we could add all these booleans as bitflags, should be more efficient.

Reply:
I'm not really satisfied with the current exception handling tbh. I will open an issue to think about different ways of how to handle all of that in the vm. I'm hoping that we can find a way to do it without all these fields at runtime.